### PR TITLE
In t5738 new

### PR DESCRIPTION
**Pull request by dineshkaushal**
submitting on behalf of siddhartha

### CODE REVIEW

**Comment by michaelDCurran** [comment]
We would like to see this abstracted out to MS Office support in general, so that it can be used in PowerPoint and Excel also.

**Comment by bdorer** [comment]
Why don't implementing this and abstract it later?

File: source/NVDAObjects/window/winword.py [hunk 1]
```diff
@@ -129,6 +131,568 @@
...
+	# Translators: a Microsoft Word Shape type
+	msoShapeNotPrimitive:_("Not supported"),
+	# Translators: a Microsoft Word Shape type
+	msoShapeFlowchartOfflineStorage:_("Offline storage flowchart symbol"),
+	# Translators: a Microsoft Word Shape type
+	msoShapeLeftRightRibbon:_("Ribbon with an arrow at both ends"),
+	# Translators: a Microsoft Word Shape type
+	msoShapeDiagonalStripe:_("Rectangle with two triangles-shapes removed; a diagonal stripe"),
+	# Translators: a Microsoft Word Shape type
+	msoShapePie:_("    Circle (pie) with a portion missing"),
```
Comment (feerrenrut) [inline source/NVDAObjects/window/winword.py]: Extra whitespace in this string

**Comment by feerrenrut** [comment]
@bdorer Because that way we would be accumulating technical debt. The code to implement this is known best now, as time passes they will forget more details about this. So the author is currently in the best position to apply this change more widely.

**Comment by michaelDCurran** [comment]
A note that this had been on hold until #7046 (Extending Excel chart support for Word/PowerPoint) had been merged. I doubt any code can really be shared, but it is a very similar design, so the idea was to see how well the chart code went first.

**Comment by dineshkaushal** [comment]
@michaelDCurran and @feerrenrut should I go ahead and implement it now?

**Comment by michaelDCurran** [comment]
It is up to you. If you feel it would be useful for people, then we'll 
take it. But we ourselves don't identify this as high priority.

**Comment by michaelDCurran** [comment]
We do want to support this in NVDA. However, this PR is very out of date now. @dineshkaushal, if you do plan to work on this based on what you have learned for Charts in MS Word, then feel free to open a new PR, including a proper description etc.

**Pull request closed without merging**